src/main/webapp/partials/instructorHelpProfile.jsp

```diff
@@ -0,0 +1,209 @@
+                <br><b>Official Email Address</b> is the email address used to contact the student.
+                <br><b>Comments</b> is any information you mentioned about the student during enrollment.
+                <br>Below this is the <b>'Mode Info'</b> section where you can see the personal description given by the student. It will look something like this -
+                <br><br>
+                <div class="modal fade" id="studentProfileMoreInfo" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                    <div class="modal-dialog modal-lg">
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                                <h4 class="modal-title">Alice Betsy's Profile - More Info</h4>
+                            </div>
+                            <div class="modal-body">
+                                <br>
+                                <%-- Note: When an element has class text-preserve-space, do not insert HTML spaces --%>
+                                <p class="text-preserve-space height-fixed-md">
+                                    Hi I am Alice Betsy! I am from Colorado, America. I am a programming and gaming enthusiast. Aspiring to become a Software Architect in a well reputed organization.
+                                </p>
+                            </div>
+                            <div class="modal-footer">
+                                <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-xs-12">
+                        <div class="panel panel-default">
+                            <div class="panel-body">
+                                <span data-toggle="modal" data-target="#studentProfileMoreInfo"
+                                    class="text-muted pull-right glyphicon glyphicon-resize-full cursor-pointer"></span>
+                                <h5>More Info </h5>
+                                <%-- Note: When an element has class text-preserve-space, do not insert HTML spaces --%>
+                                <p class="text-preserve-space height-fixed-md">Hi I am Alice Betsy! I am from Colorado, America. I am a programming and gaming enthusiast. Aspiring to become a Software Architect in a well reputed organization.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <br>You can press the <span class="text-muted glyphicon glyphicon-resize-full"></span> button on the top-right, which displays the information about the student in a modal for more readability.
```

Comment:
... ~~on~~ **in** the top-right **corner** ~~, which displays~~ **to display** the information ~~about the student~~ in a modal for ~~more~~ **better** readability.